### ISSUE TITLE
FP oelint.vars.machinespecific when using package-name

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#135]
test_git.bb
```
A_test = "foo"
```
is giving a false positive, as this implicitly defined by the filename
